prow/scripts/cluster-integration/kyma-upgrade-gardener-azure.sh

```diff
@@ -366,7 +368,7 @@ export CLUSTER_NAME="${COMMON_NAME}"
 ERROR_LOGGING_GUARD="true"
 
 export INSTALL_DIR=${TMP_DIR}
```

Comment:
you can remove this line. This was used by the Kyma-cli script for installation the binary to temporary directory.